projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -1088,21 +1088,27 @@ public static function set_availability_for_plan( $slug ) {
 	 * @param callable $render_callback The render_callback that will be called if the block is available.
 	 */
 	public static function get_render_callback_with_availability_check( $slug, $render_callback ) {
-		return function ( $prepared_attributes, $block_content ) use ( $render_callback, $slug ) {
+		return function ( $prepared_attributes, $block_content, $block ) use ( $render_callback, $slug ) {
 			$availability = self::get_cached_availability();
 			$bare_slug    = self::remove_extension_prefix( $slug );
 			if ( isset( $availability[ $bare_slug ] ) && $availability[ $bare_slug ]['available'] ) {
 				return call_user_func( $render_callback, $prepared_attributes, $block_content );
 			}
 
 			// A preview of the block is rendered for admins on the frontend with an upgrade nudge.
-			if (
-				isset( $availability[ $bare_slug ] ) &&
-				self::should_show_frontend_preview( $availability[ $bare_slug ] )
-			) {
-				$upgrade_nudge = self::upgrade_nudge( $availability[ $bare_slug ]['details']['required_plan'] );
-				$block_preview = call_user_func( $render_callback, $prepared_attributes, $block_content );
-				return $upgrade_nudge . $block_preview;
+			$upgrade_nudge = '';
```

Comment:
not sure why this is declared as an empty string first?

Reply:
Yep good catch, just bad cleanup from a refactor 👍